### PR TITLE
High impact promo - Tablet grid layout

### LINKED ISSUE #10014: High Impact Promo - Adjust Layout for mid breakpoints
**Is your feature request related to a problem? Please describe.**
We want to update the mid breakpoints for the high impact promos to improve the visual appearance. 
A large proportion of our audience use larger device types that support these mid breakpoints. 

**Describe the solution you'd like**

Update to using the latest designs: https://zpl.io/p1mneJr

**Acceptance Criteria**:
Given I am viewing a CPS or Optimo article at 600px
When I view the features and analysis section
then the high impact promo designs meet the specifications [here](https://zpl.io/r17W6ZX)

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by amoore108**
Resolves https://github.com/bbc/simorgh/issues/10014

**Overall change:**
Updates the Features and Analysis section to display in 2 columns on the tablet breakpoints

**Code changes:**
- Updates `CpsFeaturesAnalysis` to use `display: grid` to display the promos in 2 columns when the browser is within the tablet breakpoint ranges
- Updates `FrostedGlassPromo` to display the promo heights evenly when in the 2 column grid
- This is only on **Test** for now to allow UX to review

Screenshot 2022-04-19 at 2 27 10 pm

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/CpsFeaturesAnalysis/index.jsx [hunk 1]
```diff
@@ -15,8 +24,22 @@ const eventTrackingData = {
   },
 };
 
+const StoryPromoUlFeatures = styled(StoryPromoUl)`
+  ${() =>
```
Comment (HarveyPeachey) [inline src/app/containers/CpsFeaturesAnalysis/index.jsx]: Does this need an anonymous arrow function?

File: src/app/containers/CpsFeaturesAnalysis/index.jsx [hunk 2]
```diff
@@ -29,6 +52,20 @@ const StoryPromoLiFeatures = styled(StoryPromoLi)`
   &:last-child {
     padding: 0.5rem 0 0 0;
   }
+
+  ${() =>
```
Comment (HarveyPeachey) [inline src/app/containers/CpsFeaturesAnalysis/index.jsx]: Same here

Comment (amoore108) [reply src/app/containers/CpsFeaturesAnalysis/index.jsx]: Good spot, didn't realise you could write dynamic styles without a function 👍🏼

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Review by mfonofm — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Had a look on Live last week and LGTM

**Pull request merged**